frb_rust/src/handler.rs

```diff
@@ -307,6 +321,9 @@ pub trait ErrorHandler: UnwindSafe + RefUnwindSafe + Copy + Send + 'static {
     /// The default error handler.
     fn handle_error(&self, port: i64, error: Error);
 
+    /// The default panic handler.
+    fn handle_panic(&self, port: i64, error: Error);
```

Comment:
Given that `Error` enum has a sub-type `Panic`, maybe `handle_panic` can be removed and only `handle_error` is called?